index.html

```diff
@@ -25,7 +25,7 @@ <h1 style="opacity: .75">June 21</h1>
 					ontouchmove="draggingHandler(event)"
 					ontouchend="stopDraggingHandler(event)"
 				>
-					<div class="draggable-area__thumb"></div>
+					<div class="draggable-area__thumb"><div class="draggable-area__thumb--inner"></div></div>
```

Comment:
add an inner element in order to have padding on draggable thumb; to increase drag area